compiler/src/iree/compiler/Codegen/Common/GenericVectorization.cpp

```diff
@@ -155,35 +242,37 @@ void GenericVectorizationPass::runOnOperation() {
       candidates.push_back(op);
     if (vectorizePadding && enableVectorMasking && isa<tensor::PadOp>(op))
       candidates.push_back(op);
+    if (enableVectorMasking && isa<tensor::PackOp>(op))
+      candidates.push_back(op);
```

Comment:
Should we use the `if else` chain here? This can potentially push the op multiple times

Reply:
done, thanks